app/assets/javascripts/custom/content_block.js

```diff
@@ -0,0 +1,18 @@
+(function() {
+  "use strict";
+  App.ContentBlock = {
+    initialize: function() {
+      $("#site_customization_content_block_name").on({
+        change: function() {
+          if ($(this).val() === "homepage_main_content") {
+            $("#site_customization_content_block_body").addClass("html-area admin");
+            App.HTMLEditor.initialize();
+          } else {
+            $("#site_customization_content_block_body").removeClass("html-area admin");
+            App.HTMLEditor.destroy();
+          }
+        }
+      });
```

Comment:
```suggestion
      }).trigger("change");
```

This way we can remove the `ckeditor_content?` helper and the custom `_form_content_block.html.erb` view.